src/app/shared/search-box/query-item/query-items.converter.ts

```diff
@@ -78,11 +79,15 @@ export class QueryItemsConverter {
     return filters.map(filter => {
       const [collectionId, attributeId, fullCondition] = filter.split(':', 3);
       const collection = this.data.collections.find(collection => collection.id === collectionId);
-      const attribute = collection.attributes.find(attribute => attribute.id === attributeId);
+      const attribute = collection && collection.attributes.find(attribute => attribute.id === attributeId) || null;
```

Comment:
Isn't the result of this `null` by default if nothing is found? Or why is there `|| null` at the end?